README.md

```diff
@@ -56,6 +57,11 @@ The search bar uses `Plasma Search`, which is the same search provider used in K
   <pre>kpackagetool5 -t Plasma/Applet --install</pre>
 </p>
 
+<p align="center">Now make sure you have plasma-framework package installed by execute the following command:</p>
+<p align="center">
+  <pre>sudo apt install plasma-framework</pre>
+</p>
```

Comment:
* Put this part in the `Prerequisites:` section below or within `* Plasma-Framework Package`